src/pages/ReviewPage.test.tsx

```diff
@@ -35,5 +38,7 @@ it(`renders ReviewPage with votes`, () => {
       },
     }
   )
+
+  expect(badDOMIdRegexp.test(container.outerHTML)).toBeFalsy()
```

Comment:
I think this test is unnecessary because:
- this test will never be run in production (and thus never catch a production error).
- this test doesn't cover all app code where numeric `id` values may exist.
- we have prefixed the id value in the template… so there is no chance of this occurring.
- it is not testing anything which a user will encounter.

Do you agree?

Reply:
oh excellent point, but I think I want to resolve it a different way: let's introduce some contest IDs that do start with a digit in `electionSample.json`. Then this test *will* be useful at ensuring that our code protects against numerical contest IDs, which we will see in config files produced by other scripts/code sometimes not our own.

What do you think?

Reply:
wait, actually, I already did this :) My past self is smarter than my today self. 

I changed `measure-102` to `102` for the sole purpose of making sure this test would catch any code change that fails to protect DOM node IDs.